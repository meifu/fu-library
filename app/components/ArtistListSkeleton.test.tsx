import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';

import ArtistListSeleton from './ArtistListSkeleton';

describe('Component: Title', () => {
  it('render Title correctly', () => {
    const tree = renderer.create(<ArtistListSeleton />);
    expect(tree).toMatchSnapshot();
  });
});
