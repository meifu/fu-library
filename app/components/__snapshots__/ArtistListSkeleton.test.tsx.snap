// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Title render Title correctly 1`] = `
<div
  className="MuiStack-root css-1ty7g7t-MuiStack-root"
>
  <span
    className="MuiSkeleton-root MuiSkeleton-rounded MuiSkeleton-wave css-1hirv1y-MuiSkeleton-root"
    style={
      {
        "height": 60,
        "width": "100%",
      }
    }
  />
  <span
    className="MuiSkeleton-root MuiSkeleton-rounded MuiSkeleton-wave css-1hirv1y-MuiSkeleton-root"
    style={
      {
        "height": 60,
        "width": "80%",
      }
    }
  />
  <span
    className="MuiSkeleton-root MuiSkeleton-rounded MuiSkeleton-wave css-1hirv1y-MuiSkeleton-root"
    style={
      {
        "height": 60,
        "width": "100%",
      }
    }
  />
  <span
    className="MuiSkeleton-root MuiSkeleton-rounded MuiSkeleton-wave css-1hirv1y-MuiSkeleton-root"
    style={
      {
        "height": 60,
        "width": "90%",
      }
    }
  />
  <span
    className="MuiSkeleton-root MuiSkeleton-rounded MuiSkeleton-wave css-1hirv1y-MuiSkeleton-root"
    style={
      {
        "height": 60,
        "width": "100%",
      }
    }
  />
  <span
    className="MuiSkeleton-root MuiSkeleton-rounded MuiSkeleton-wave css-1hirv1y-MuiSkeleton-root"
    style={
      {
        "height": 60,
        "width": "90%",
      }
    }
  />
  <span
    className="MuiSkeleton-root MuiSkeleton-rounded MuiSkeleton-wave css-1hirv1y-MuiSkeleton-root"
    style={
      {
        "height": 60,
        "width": "100%",
      }
    }
  />
</div>
`;
